save-cloud-charts/save-cloud/templates/fs-storage-pvc.yaml

```diff
@@ -1,21 +1,24 @@
+{{- range tuple .Values.backend.name .Values.sandbox.name }}
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   labels:
-    io.kompose.service: save-fs-storage
-  {{- with .Values.storage.annotations  }}
+    io.kompose.service: {{ . }}-fs-storage
+  {{- with $.Values.storage.annotations  }}
```

Comment:
Default context (`.`) becomes overridden by `range`, so `$` can be used to access root context